15_1_Starting_the_Regtest.md

```diff
@@ -0,0 +1,31 @@
+# 15.1: Starting the Regtest
+
+> **NOTE:** This is a draft in progress, so that I can get some feedback from early reviewers. It is not yet ready for learning.
+
+This document explains how to build a Regtest (Regression Test) by hand to be able to develop new applications without the need to interact with other peers and blocks.
```

Comment:
`how to set up a bitcoin node locally using the regtest (regression test) network in order to develop new applications`[...]